portfolio/src/main/java/com/google/sps/translation/TranslationProvider.java

```diff
@@ -17,22 +17,63 @@
 import com.google.cloud.translate.Translate;
 import com.google.cloud.translate.TranslateOptions;
 import com.google.cloud.translate.Translation;
+import com.google.cloud.translate.Language;
+import com.google.sps.data.Comment;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
-/** Class that allows to translate text. */
+/** Class that allows to translate. */
 public class TranslationProvider {
+  public static final String DEFAULT_TARGET_LANGUAGE = "en";
+  private Translate translate;
+  private Set<String> supportedLanguageCodes;
```

Comment:
TODO: use this list to construct list of languages on the frontend side (instead of hardcoding)

Reply:
Good idea. I'd consider doing this in a separate PR though.

Reply:
Yeah, that's why it is a TODO. :)